article-details.blade.php

```diff
@@ -95,21 +95,22 @@ <h2 class="article-date">{{article date here}} منذ ٢ أيام</h2>
                       <!-- article content here -->
                       <div class="info-wrapper">
                         <!-- article html content here -->
-                        اتبينسم يىسشتىينتسش ىلتيسش ىتليسش ىتل ىشتل ىشتلي سىشتمليسش اتبينسم يىسشتىينتسش ىلتيسش ىتليسش ىتل ىشتل ىشتلي سىشتمليسش اتبينسم يىسشتىينتسش ىلتيسش ىتليسش ىتل ىشتل ىشتلي سىشتمليسش اتبينسم يىسشتىينتسش ىلتيسش ىتليسش ىتل ىشتل ىشتلي سىشتمليسش اتبينسم يىسشتىينتسش ىلتيسش ىتليسش ىتل ىشتل ىشتلي سىشتمليسش اتبينسم يىسشتىينتسش ىلتيسش ىتليسش ىتل ىشتل ىشتلي سىشتمليسش ر ر ر ر ر  اتبينسم يىسشتىينتسش ىلتيسش ىتليسش ىتل ىشتل ىشتلي سىشتمليسشر اتبينسم يىسشتىينتسش ىلتيسش ىتليسش ىتل ىشتل ىشتلي  سىشتملي سشات بينسم يىس شتىي نتسش ىلتيسش ىتليسش ىتل ىشتل ىشتلي سىشتمليسش
-                        اتبينسم يىسشتىينتسش ىلتيسش ىتليسش ىتل ىشتل ىشتلي سىشتمليسش اتبينسم يىسشتىينتسش ىلتيسش ىتليسش ىتل ىشتل ىشتلي سىشتمليسش اتبينسم
+                        {{ $content['article']['content'] }}
                       </div>
                       <div class="article-speaks-about">
                         <div class="header">هذا المقال يتحدث عن</div>
                         <ul class="celebrities-pics">
                          <!-- related influencers here -->
+                         @foreach ($content['article']['influencers'] as $influencer)
                           <li>
                             <!-- slug should be in the href -->
-                            <a href="/influencers/{{slug here}}">
+                            <a href="/-/influencers/{{ $influencer['alias'] }}">
```

Comment:
links should be to the website and not to the data (JSON). See https://github.com/Trelllis/najem-seo-templates/issues/4